rclcpp/test/test_mapped_ring_buffer.cpp

```diff
@@ -0,0 +1,154 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include <rclcpp/mapped_ring_buffer.hpp>
+
+/*
+Tests get_copy and pop on an empty mrb.
+*/
+TEST(test_mapped_ring_buffer, empty) {
```

Comment:
`test_mapped_ring_buffer` is a class name and should therefore be camel case. Same for the other tests.